### DISCUSSION TITLE
Is there any version of textadept on the web?

### DISCUSSION THREAD

**Discussion by ghost** [orbitalquark/textadept#316] [Q&A] [answered]
Hi all.

Is there any version of textadept on the web? Where can I use it in the browser without having to download a zip file or install it on my machine in textadept?

**Comment by orbitalquark** [comment, marked as answer]
No, there is not :( Textadept is a desktop application.

**Reply by ghost** [reply]
Hi orbitalquark, 

> No, there is not :( Textadept is a desktop application.

thank you for feedback.
